package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@across-protocol/sdk-v2",
   "author": "UMA Team",
-  "version": "0.15.22",
+  "version": "0.15.23",
```

Comment:
This should probably be a bump in the minor, given that we're deprecating something.